pkg/connectors/policymanager/clients/policymanager_openapi.go

```diff
@@ -41,19 +42,20 @@ func NewOpenAPIPolicyManager(name string, connectionURL string, connectionTimeou
 	return &openAPIPolicyManager{
 		name:   name,
 		client: apiClient,
+		log:    log,
 	}, nil
 }
 
 func (m *openAPIPolicyManager) GetPoliciesDecisions(in *policymanager.GetPolicyDecisionsRequest, creds string) (*policymanager.GetPolicyDecisionsResponse, error) {
 	resp, r, err := m.client.DefaultApi.GetPoliciesDecisionsPost(context.Background()).XRequestCred(creds).PolicyManagerRequest(*in).Execute()
 	// resp, r, err := m.client.DefaultApi.GetPoliciesDecisions(context.Background()).Input(*in).Creds(creds).Execute()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error when calling `DefaultApi.GetPoliciesDecisions``: %v\n", err)
-		fmt.Fprintf(os.Stderr, "Full HTTP response: %v\n", r)
+		m.log.Error().Err(err).Msg("error when calling `DefaultApi.GetPoliciesDecisions`")
```

Comment:
Please add the datasetid and uuid

Reply:
Added datasetid. Same comment about uuid.